eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -180,10 +187,13 @@ stages:
                           WorkingDirectory: $(System.DefaultWorkingDirectory)
                           TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
                           TargetDocRepoName: ${{parameters.TargetDocRepoName}}
-                          PRBranchName: 'smoke-test-rdme'
-                          ArtifactName: ${{parameters.ArtifactName}}
+                          PRBranchName: docs-${{artifact.name}}-$(Build.BuildId)
```

Comment:
why isn't this branch called onboarding like the others or why isn't the others called docs like this one?

Reply:
Yeah. Thank you for the catch. Will rename.